File: sources/Android/android-communications/library/src/sdk/java/com/polar/sdk/api/model/LedConfig.kt
package com.polar.sdk.api.model

data class LedConfig(
    val sdkModeLedEnabled: Boolean = true,
    val ppiModeLedEnabled: Boolean = true
) {
    companion object {
        const val LED_CONFIG_FILENAME = "/LEDCFG.BIN"
        const val LED_ANIMATION_DISABLE_BYTE: Byte = 0x00
        const val LED_ANIMATION_ENABLE_BYTE: Byte = 0x01
    }
}

File: sources/iOS/ios-communications/Sources/PolarBleSdk/sdk/api/model/LedConfig.swift
//  Copyright © 2023 Polar. All rights reserved.

import Foundation

public struct LedConfig {
    let sdkModeLedEnabled: Bool
    let ppiModeLedEnabled: Bool

    static let LED_CONFIG_FILENAME = "/LEDCFG.BIN"
    static let LED_ANIMATION_DISABLE_BYTE: UInt8 = 0x00
    static let LED_ANIMATION_ENABLE_BYTE: UInt8 = 0x01
    
    public init(sdkModeLedEnabled: Bool, ppiModeLedEnabled: Bool) {
        self.sdkModeLedEnabled = sdkModeLedEnabled
        self.ppiModeLedEnabled = ppiModeLedEnabled
    }
}